extensions/applicationinsights-clickanalytics-js/src/handlers/DomContentHandler.ts

```diff
@@ -75,21 +71,24 @@ export class DomContentHandler implements IContentHandler {
                 }
             } else {
                 // traverse up the DOM to find the closest parent with data-* tag defined
-                contentElement = walkUpDomChainWithElementValidation(element, this._isTracked, dataTagPrefix);
-                elementContent = extend(elementContent, this._populateElementContentwithDataTag( contentElement, element, dataTagPrefix, parentDataTagPrefix));
+                //contentElement = walkUpDomChainWithElementValidation(element, this._isTracked, dataTagPrefix);
```

Comment:
Don't we still want to do this -- just not for the id

Reply:
We are doing this only when user has specified the parentDataTag. This tag does two things. IF present it auto populates parent content and also used as an identifier to stop traversing up the DOM to capture content name and value of elements when encountered with this tag.